app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -229,48 +174,40 @@ class TeamsAndRepositoriesConnector @Inject()(http: HttpClient, servicesConfig:
   }
 
   def allTeams(implicit hc: HeaderCarrier): Future[Seq[Team]] =
-    http.GET[Seq[Team]](url"$teamsAndServicesBaseUrl/api/teams?includeRepos=true")
+    http.GET[Seq[Team]](url"$teamsAndServicesBaseUrl/api/v2/teams")
+
+  def repositoriesForTeam(teamName: TeamName, includeArchived: Option[Boolean] = None)(implicit hc: HeaderCarrier): Future[Seq[GitRepository]] = {
+    val url = includeArchived.fold( url"$teamsAndServicesBaseUrl/api/v2/repositories?team=${teamName.asString}")(arc =>  url"$teamsAndServicesBaseUrl/api/v2/repositories?team=${teamName.asString}&archived=$arc")
```

Comment:
or just `archived=$includeArchived` and let the interpolator remove the param if `None`